### ISSUE TITLE
[Feature Request] Decimal precision in currency conversion

### ISSUE THREAD

**Issue by yauyenching** [sal0max/currencies#61]
Hi! Love your app and how minimalistic and clean the UI is. 🙂 I also +1 the existing feature request of adding more favourite currencies to the home screen!

As for my request, I love if there would be decimal precision for up to 4-6 digits (perhaps as a user setting).

For example, when I convert 1 MYR to SGD, I see 1 -> 0.3, but I rather see 1 -> 0.295707. Thanks so much!

**Comment by sal0max** [comment]
Thanks for the kind words.

- The precision in the main sceen is always two digits. So in your case, `1.00 MYR` was `0.30 SGD`. You see, the scope of the app is to be a practical companion on you vacation trips. Your budget won't be affected by differences in the tenths.
- However, you already get a higher precision (dynamically adjusted) in the exchange rates history screen.

Is this good enough for you, or what is your use case?

**Comment by 825i** [comment]
I completely agree that it's good enough how it is. However, it might be nice as a toggle feature (if you or anyone else feels like it). At least sadly Open Exchange Rates doesn't support history.

**Comment by Xemnarth** [comment]
+1 — I’d also find this feature really useful, especially for low-value currencies. Thanks for considering!
